src/main/java/com/wavesplatform/wavesj/Node.java

```diff
@@ -540,24 +465,24 @@ public TransactionInfo getTransactionInfo(Id txId) throws IOException, NodeExcep
                 .readValue(asJson(request).get(0));
     }
 
-    public TransactionStatus getTransactionStatus(Id id) throws IOException, NodeException {
-        return asType(get("/transactions/status").addParameter("id", id.toString()),
+    public TransactionStatus getTransactionStatus(Id txId) throws IOException, NodeException {
+        return asType(get("/transactions/status").addParameter("id", txId.toString()),
                 TypeRef.TRANSACTIONS_STATUSES).get(0);
     }
 
-    public List<TransactionStatus> getTransactionsStatuses(List<Id> ids) throws IOException, NodeException {
+    public List<TransactionStatus> getTransactionsStatuses(List<Id> txIds) throws IOException, NodeException {
         ObjectNode jsonBody = JSON_MAPPER.createObjectNode();
         ArrayNode jsonIds = jsonBody.putArray("ids");
-        ids.forEach(id -> jsonIds.add(id.toString()));
+        txIds.forEach(id -> jsonIds.add(id.toString()));
         StringEntity body = new StringEntity(JSON_MAPPER.writeValueAsString(jsonBody), StandardCharsets.UTF_8);
 
         return asType(post("/transactions/status")
                 .addHeader("Content-Type", "application/json")
                 .setEntity(body), TypeRef.TRANSACTIONS_STATUSES);
     }
 
-    public List<TransactionStatus> getTransactionsStatuses(Id... ids) throws IOException, NodeException {
-        return getTransactionsStatuses(new ArrayList<>(Arrays.asList(ids)));
+    public List<TransactionStatus> getTransactionsStatuses(Id... txIds) throws IOException, NodeException {
```

Comment:
rename to getTransactionsStatus

Reply:
> rename to getTransactionsStatus

Why? Statuses is proper naming for list

Reply:
> Why? Statuses is proper naming for list

Just for consistency with other functions and name of the endpoint, `/transactions/status`.